hieradata/hosts/test4.yaml

```diff
@@ -0,0 +1,140 @@
+      - "mw9.miraheze.org:443"
+      - "mw10.miraheze.org:443"
+      - "mw11.miraheze.org:443"
+    parent_is_proxy: 'false'
+    round_robin: 'strict'
+
+role::trafficserver::default_lua_script: 'default'
+
+role::trafficserver::enable_compress: false
+
+role::trafficserver::inbound_tls_settings:
+  cert_path: '/etc/ssl/localcerts/'
+  private_key_path: '/etc/ssl/private/'
+  common:
+    cipher_suite: '-ALL:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-RSA-AES128-GCM-SHA256'
+    cipher_suite_tlsv1_3: 'TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256'
+    groups_list: 'X25519:prime256v1'
+    enable_tlsv1: 0
+    enable_tlsv1_1: 0
+    enable_tlsv1_2: 1
+    enable_tlsv1_3: 1
+  max_record_size: 16383
+  session_cache: 2
+  session_cache_timeout: 86400
+  session_cache_auto_clear: 1
+  session_cache_size: 512000
+  session_cache_buckets: 32768
+  session_ticket_enable: 0
+  load_elevated: 1
+  do_ocsp: 0
+  ssl_handshake_timeout_in: 60
+  prioritize_chacha: 1
+
+role::trafficserver::outbound_tls_settings:
+  common:
+    cipher_suite: '-ALL:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384'
+    cipher_suite_tlsv1_3: 'TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256'
+    enable_tlsv1: 0
+    enable_tlsv1_1: 0
+    enable_tlsv1_2: 1
+    enable_tlsv1_3: 1
+  verify_origin: true
+  cacert_dirpath: '/etc/ssl/certs'
+  cacert_filename: 'ca-certificates.crt'
+
+role::trafficserver::log_formats:
+  - name: backend
+    # TTFetchHeaders is the amount of time between the first origin server
+    # connection attempt (or shared session attached in case of connection
+    # reuse) and when the first byte is received from the origin.
+    # ClientTTFB is the time spent between the very beginning of this session
+    # and when the response header write to the client starts.
+    format: 'Date:%<cqtd> Time:%<cqtt> ConnAttempts:%<sca> ConnReuse:%<sstc> TTFetchHeaders:%<{TS_MILESTONE_SERVER_FIRST_READ-TS_MILESTONE_SERVER_FIRST_CONNECT}msdms> ClientTTFB:%<{TS_MILESTONE_UA_BEGIN_WRITE-TS_MILESTONE_SM_START}msdms> CacheReadTime:%<{TS_MILESTONE_CACHE_OPEN_READ_END-TS_MILESTONE_CACHE_OPEN_READ_BEGIN}msdms> CacheWriteTime:%<{TS_MILESTONE_CACHE_OPEN_WRITE_END-TS_MILESTONE_CACHE_OPEN_WRITE_BEGIN}msdms> TotalSMTime:%<{TS_MILESTONE_SM_FINISH-TS_MILESTONE_SM_START}msdms> OriginServer:%<shn> OriginServerTime:%<stms> CacheResultCode:%<crc> CacheWriteResult:%<cwr> ReqMethod:%<cqhm> RespStatus:%<pssc> OriginStatus:%<sssc> ReqURL:%<cquuc> ReqHeader:User-Agent:%<{User-agent}cqh> ReqHeader:Host:%<{Host}cqh> ReqHeader:X-Client-IP:%<{X-Client-IP}cqh> ReqHeader:Cookie:%<{Cookie}cqh> BerespHeader:Set-Cookie:%<{Set-Cookie}ssh[0:16]> BerespHeader:Cache-Control:%<{Cache-Control}ssh> BerespHeader:Connection:%<{Connection}ssh> RespHeader:X-Cache-Int:%<{X-Cache-Int}psh> RespHeader:Backend-Timing:%<{Backend-Timing}psh>'
```

Comment:
Note: at the moment X-Real-IP is used on destination servers for determining the correct source IP.